src/mysass.scss

```
@@ -0,0 +1,41 @@
+
+$myColor: red;
+$reuseColor:blueviolet;
+$impTextColor:rgb(47, 29, 56);
+$myStyling: rgb(184, 204, 95);
+
+.navigation-title {
+  color: $myColor;
+}
+
+.reuse{
+ color: $reuseColor;
+}
+
+@mixin important-text{
+  color: $impTextColor;
+  font-size: 20px;
+  font-weight: bold;
+  border: 1px solid $reuseColor;
+  
+}
+.container {
+    @include important-text;
+    background-color: $myStyling;
+  }
+
+.navigation {
+    ul {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+    }
+    li {
+      display: inline-block;
+    }
+    a {
+      display: block;
+      padding: 6px 12px;
+      text-decoration: none;
+    }
+  }
```

Tidy up the code a bit
1. use meaningful names - its okay to take some time and come up with a meaningful name
2. keep varaibles and mixins on the top
3. pay attention to indentation

Example
```suggestion

$highlight: red;
$secondary: blueviolet;
$primary: rgb(47, 29, 56);
$container-background: rgb(184, 204, 95);

@mixin default-font {
  color: $primary;
  font-size: 20px;
  font-weight: bold;
  border: 1px solid $secondary;
  
}

.navigation-title {
  color: $highlight;
}

.reuse{
  color: $secondary;
}

.container {
    @include default-font;
    
    background-color: $container-background;
  }

.navigation {
    ul {
      margin: 0;
      padding: 0;
      list-style: none;
    }
    
    li {
      display: inline-block;
    }
    
    a {
      display: block;
      padding: 6px 12px;
      text-decoration: none;
    }
  }
```